server/methods.js

```diff
@@ -675,4 +677,87 @@ Meteor.methods({
 
     return Accounts.createUser({ email, password, profile: { name: profileName } });
   },
+  createPDF(slug) {
+    let questionList = [];
+    let answerList = [];
+    let boldFont = 'Helvetica-Bold';
+    let normalFont = 'Helvetica';
+    let headingFontSize = 28;
+    let normalFontSize = 14;
+    let doc = new PDFDocument({ size: 'A4', margin: 50 });
+
+    if (!doc) {
+      console.log('Not able to initialize PDFKit object');
+      return false;
+    }
+
+    let currentInstance = Instances.findOne({ slug: slug });
+    let currentUser = Meteor.users.findOne({ _id: this.userId }).emails[0].address;
+    if(currentUser !== currentInstance.admin) {
+      return false;
+    }
+    let randomNum = Math.floor(Math.random() * (10000 - 1 + 1)) + 1;
+    let filename = 'Instance_archive_' + currentInstance.tablename + '_' + randomNum + '.pdf';
+    let filepath = process.env.PWD + '/' + filename;
+    doc.fontSize(headingFontSize);
+    doc.text(`This is the archive of ${currentInstance.tablename}`, {underline: true, });
+    doc.moveDown();
+    let questionsOnCurrentInstance = Questions.find({ instanceid: currentInstance._id });
+    let noOfQuestionsOnCurrentInstance = Questions.find({ instanceid: currentInstance._id }).count();
+    let answersOnCurrentInstance = Answers.find({ instanceid: currentInstance._id });
+    answersOnCurrentInstance.forEach(function(ans) {
+      answerList.push(ans);
+    });
+    doc.fontSize(normalFontSize);
+    doc.text(`Author: ${currentInstance.author}`, 20, doc.y, {align: 'left'});
+    doc.text(`Description of instance: ${currentInstance.description}`, 20, doc.y, {align: 'left'});
```

Comment:
@singhsanket143 

can you replace?

`Description of instance` > `Description Of The Instance`
`Max Question Allowed` > `Max. Questions Allowed`
`Max Response Allowed` > `Max. Responses Allowed`
`Total Questions Posted Till Now` > `Total Questions Posted Until Now`
`List of replies` > `List Of Replies`

Reply:
@peter-hank Updated the file